src/Promenade.Service/Promenade.Service.csproj

```diff
@@ -12,6 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Ops.Service\Ops.Service.csproj" />
```

Comment:
Promenade project should not include references to Ops projects. It seems like there are some crossed wires here.